src/Microsoft.Extensions.SecretManager.Tools/Internal/ProjectIdResolver.cs

```diff
@@ -35,32 +36,37 @@ public string Resolve(string project, string configuration = Constants.DefaultCo
             var outputFile = Path.GetTempFileName();
             _tempFiles.Add(outputFile);
 
-            var commandOutput = new List<string>();
-            var commandResult = Command.CreateDotNet("msbuild",
-                new[] {
-                    targetFile,
-                    "/nologo",
-                    "/t:_FindUserSecretsProperty",
-                    $"/p:Project={projectFile}",
-                    $"/p:OutputFile={outputFile}",
-                    $"/p:Configuration={configuration}"
-                })
-                .CaptureStdErr()
-                .CaptureStdOut()
-                .OnErrorLine(l => commandOutput.Add(l))
-                .OnOutputLine(l => commandOutput.Add(l))
-                .Execute();
-
-            if (commandResult.ExitCode != 0)
+            var args = new[]
             {
-                _logger.LogDebug(string.Join(Environment.NewLine, commandOutput));
-                throw new GracefulException(Resources.FormatError_ProjectFailedToLoad(projectFile));
+                "msbuild",
+                targetFile,
+                "/nologo",
+                "/t:_FindUserSecretsProperty",
+                $"/p:Project={projectFile}",
```

Comment:
This is just moved code (not new), but is there a bug here if there are spaces in anything? Are spaces valid in any of these values? (Should log this as a separate bug, assuming it's actually a bug.)